UnityProject/Assets/Scripts/Closets/ClosetControl.cs

```diff
@@ -152,26 +151,41 @@ public override void Interact(GameObject originator, Vector3 position, string ha
             //FIXME this should be rewritten to net messages, see i.e. TableTrigger
             if (Input.GetKey(KeyCode.LeftControl))
                 return;
-
             if (PlayerManager.PlayerInReach(transform))
             {
-                if (IsClosed)
+                //we round to int because this wasn't done where it needs to be done on the player
+                Vector3 playerPos = new Vector3(Mathf.RoundToInt(PlayerManager.LocalPlayer.transform.position.x), Mathf.RoundToInt(PlayerManager.LocalPlayer.transform.position.y), PlayerManager.LocalPlayer.transform.position.z);

```

Comment:
Couldn't you use vector3int here?

Reply:
okey, that leaves 2 more things:
1. What about above vector3int thingy?
2. Did you multiplayer test it?